### PR TITLE
Add find_tabs.py file to help with testgrid config

### PR DESCRIPTION
**Pull request by fejta**
For the time being we still need to process this in order to push the config, but this saves a lot of typing and hopefully can be used by other people.

### CODE REVIEW

File: testgrid/find_tabs.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+import re
+import subprocess
+import sys
+
+# The value for each of these represents a testgrid dashboard
+# group(1) represents the tab name
+MATCHES = [
+    (r'kubernetes-e2e-gce-(gci.*)', 'gci'),
+    (r'kubernetes-e2e-(.*-trusty.*)', 'trusty'),  # contains gce
+    (r'kubernetes-e2e-(.*-upgrade.*)', 'upgrade'),  # contains gke
```
Comment (spxtr) [inline testgrid/find_tabs.py]: I think the kubectl-skew tests should be in upgrade as well.

File: testgrid/find_tabs.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+          mat = regex.match(path)
+          if not mat:
+              continue
+          tab_name = mat.group(1)
+          yield dash, tab_name, path
+          break
+
+
+def fill_dashboards(tabs):
+    """Returns {dash: (tab, path)}, set([path]) tuple from found tabs."""
```
Comment (apelisse) [inline testgrid/find_tabs.py]: You meant `dash: [(tab, path)]`?

**Comment by apelisse** [comment]
LGTM

File: testgrid/find_tabs.py (hunk 2 above)
Comment (fejta) [inline testgrid/find_tabs.py]: Yes, thanks!

File: testgrid/find_tabs.py (hunk 1 above)
Comment (fejta) [inline testgrid/find_tabs.py]: Done

**Pull request merged**